doc/source/deployment/setup-deployer.rst

```diff
@@ -87,7 +87,20 @@ There are several ways to install the SUSE Containerized OpenStack software.
    b. sudo zypper install socok8s (installs to /usr/share/socok8s)
 
 
```

Comment:
@jgu17 if we add the required sw in this section, should we remove the previous section:
https://github.com/SUSE-Cloud/socok8s/blob/master/doc/source/deployment/setup-deployer.rst#base-software

Reply:
yes. good catch. Fixed